test/hack/bin/check-opsrecipes.sh

```diff
@@ -57,7 +90,7 @@ main() {
     ########################
 
     # Retrieve list of opsrecipes
-    mapfile -t opsRecipes < <(listOpsRecipes)
+    mapfile -t opsRecipes < <(listOpsRecipes $runInCi)
```

Comment:
:heart: quotes :heart: :grin: 
```suggestion
    mapfile -t opsRecipes < <(listOpsRecipes "$runInCi")
```